runtime/j9vm/j7vmi.c

```diff
@@ -380,7 +380,7 @@ java_lang_J9VMInternals_doPrivilegedWithException(JNIEnv* env)
 	return cached;
 }
 
-
+/* Do not remove: this method is used in Windows Java 8 vmfarm builds. */
```

Comment:
Not sure "used" is accurate. It should never be called, but there is code that links to it.

Reply:
Pls use "Windows IBM Java 8" or similar. I don't vmfarm should be used, the build platform may change (perhaps it will be jenkins in the future) but the link issue will persist unless the IBM Java 8 JCL code is modified.